java/arcs/core/host/AbstractArcHost.kt

```diff
@@ -230,17 +277,16 @@ abstract class AbstractArcHost(vararg initialParticles: ParticleRegistration) :
     /**
      * Return an instance of [EntityHandleManager] to be used to create [Handle]s.
      */
-    open fun entityHandleManager(): EntityHandleManager = EntityHandleManager(
-        HandleManager(platformTime())
-    )
+    open fun entityHandleManager(): EntityHandleManager =
```

Comment:
can this be an `open val`? 

Also, this will make a new manager every time it's called.. is that intended?

Reply:
Good question, it probably isn't needed to be new everytime.